### PR TITLE
Adding absolute Url to clippath

### PR DESCRIPTION
**Pull request by csxphil**
Allows for absoluteUrl to be supplied to the clippath

Developers in frameworks such as angularjs have issues with a relative url supplied. Specify a string url path as an option. 

The default value is a blank string, so not using this option will cause no effect for normal usage.

### CODE REVIEW

File: lib/drawer/index.js [hunk 1]
```diff
@@ -29,8 +29,8 @@ export default (svg, dimensions, scales, configuration) => {
 
     const dropsContainer = svg.append('g')
         .classed('drops-container', true)
-        .attr('clip-path', 'url(#drops-container-clipper)')
-        .style('filter', 'url(#metaballs)');
+        .attr('clip-path', 'url('+configuration.absoluteUrl+'#drops-container-clipper)')
```
Comment (jpetitcolas) [inline lib/drawer/index.js]: Can you use ES6 string concatenation?

``` js
`url(${configuration.absoluteUrl}#drops-container-clipper)`
```

**Comment by jpetitcolas** [comment]
Absolute URL is current page URL, isn't it? So, can't we retrieve it automatically from `window.location`?

**Comment by csxphil** [comment]
Added ES6.

I used a string as the value as with angular you use $window instead of window, and uses a different attribute for getting location. Other platforms might differ further so I thought it more flexible to allow the user to specify the value of the absoluteUrl within the scope of where the chart will be created.

**Comment by jpetitcolas** [comment]
@csxphil, any news about it? :)

**Comment by csxphil** [comment]
So I think I'll close this one, the real issue appeared to be that I had used a page template with the base tag set to '/' so it was losing context. While the issue might have continued, Angular has a workaround to ignore the base tag, which allowed SVG to relative URLs to work. I'll be closing this pull request and switching back to your library, as my fork with this feature implemented is no longer needed.

**Comment by csxphil** [comment]
And thank you for the consideration and help!

**Pull request closed without merging**